src/components/ProjectSettingsTabs/Header.tsx

```diff
@@ -92,7 +94,6 @@ export const Header: React.FC<Props> = ({
           });
 
     save(newData, mapRouteToSection[tab]);
-    saveTab(tab);
```

Comment:
This saveTab function is used to disable the save button; we only want to call it if the save operation succeeded